### PR TITLE
[IMP] mail: do not exclude self from partner search

### PR DESCRIPTION
**Pull request by oomsveta**
Do not exclude the current partner from the partner search so that you can easily start a chat with yourself.

Task-2844917

### CODE REVIEW

**Comment by alexkuhn** [comment]
runbot's red due to (IMO badly written) test to adapt

**Comment by oomsveta** [comment]
> runbot's red due to (IMO badly written) test to adapt

@alexkuhn test (hopefully) fixed 👌

File: addons/mail/static/tests/qunit_suite_tests/webclient/commands/mail_providers_tests.js [hunk 1]
```diff
@@ -23,20 +23,8 @@ QUnit.module('mail', {}, function () {
...
-            { name: "Partner 1", email: "p1@odoo.com" },
-            { name: "Partner 2", email: "p2@odoo.com" },
-            { name: "Partner 3", email: "p3@odoo.com" },
-        ]);
-        pyEnv['res.users'].create([
-            { name: "User 1", partner_id: resPartnerId1 },
-            { name: "User 2", partner_id: resPartnerId2 },
-            { name: "User 3", partner_id: resPartnerId3 },
-        ]);
+    QUnit.only('open the chatWindow of a user from the command palette', async function (assert) {
```
Comment (alexkuhn) [inline addons/mail/static/tests/qunit_suite_tests/webclient/commands/mail_providers_tests.js:26]: Remove `.only`

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**